product_docs/docs/postgres_for_kubernetes/1/iron-bank.mdx

```diff
@@ -33,19 +33,25 @@ Specifically, you will need to use your
 [registry1](https://registry1.dso.mil/harbor/projects)
 credentials to pull images.
 
-To find the desired operator image, we recommend to use the search tool to look
+To find the desired operator or operand images, we recommend to use the search tool to look
 with the string `edb`, and filter by `Tags`, looking for `stable`, as shown in
-the image. From there, you can get the instruction to pull the image, for
-example using Docker:
+the image. From there, you can get the instruction to pull the image:
 
 ![pulling-ironbank-images](./images/ironbank/pulling-the-image.png)
 
+For example, to pull the latest EPAS16 operand from Ironbank, you can run:
```

Comment:
Minor nit: this is explicitly pulling version 16 of EPAS. In Docker `latest` is a specific tag the always pulls the latest version.

Reply:
Yep, I say the latest version of EPAS16 not of EPAS, so I am not trying to token `latest` here.

Reply:
I went ahead and removed the word "latest" just to minimize confusion.